assets/js/shortcake-bakery.js

```diff
@@ -30,8 +30,21 @@
 				parentWidth = parent.innerWidth;
 			}
 
-			var trueHeight = el.data('true-height') ? el.data('true-height') : 360;
-			var trueWidth = el.data('true-width') ? el.data('true-width') : 640;
+			var trueHeight = 360,
+				trueWidth = 640;
+
+			if ( el.data('true-height') ) {
+				trueHeight = el.data('true-height').replace( 'px', '' );
+			} else if ( el.attr('height') ) {
+				trueHeight = el.attr('height').replace( 'px', '' );
+			}
+
+			if ( el.data('true-width') ) {
+				trueWidth = el.data('true-width').replace( 'px', '' );
+			} else if ( el.attr('width') ) {
+				trueWidth = el.attr('width').replace( 'px', '' );
```

Comment:
What happens when width is set to a value like `100%`? Does that need to be handled specially? I think cases like that were likely behind the idea of using a data attribute to store "true" width and height.